src/project/settings/local.py.dist

```diff
@@ -8,7 +8,8 @@
 
 SECRET_KEY = 'super-hot-secret-123456789'
 
-# Create the obsevatory layer, update next next line and uncomment it
+# Create the obsevatory layer, update the two next lines and uncomment them
 # TROPP_OBSERVATORY_LAYER_PK = 1
+# TROPP_OBSERVATORY_ID = 20
```

Comment:
Add default value or add waring at startup like for obs id

Reply:
Checks and warnings have been added to `terra-opp`. Is it enough?

Reply:
Yes!